rust/windows-client/src-tauri/Cargo.toml

```diff
@@ -9,15 +9,29 @@ edition = "2021"
 tauri-build = { version = "1.5", features = [] }
 
 [dependencies]
+anyhow = { version = "1.0" }
+clap = { version = "4.4", features = ["derive",  "env"] }
 connlib-client-shared = { workspace = true }
+directories = "5.0.1"
+firezone-cli-utils = { workspace = true }
+keyring = "2.0.5"
+# Not the newest `open` but this is what Tauri uses internally already
+open = "3.2.0"
```

Comment:
I assume it would compile a tiny bit faster if we only have to compile one version of a given crate.

`open` looks like a pretty thin wrapper around `std::process` so I'd be okay updating it. Or just skimming the code and replacing it with `std::process`.

Reply:
Resolved in 437b0d5e - Tauri already had an API for it, so I removed our direct dep on `open`